scripts/plots/analyses/exevs_analyses_grid2obs_plots.sh

```diff
@@ -11,10 +11,8 @@ set -x
 
 # Set up initial directories and initialize variables
 
-mkdir -p $DATA/plots
 mkdir -p $DATA/plots/logs
 export LOGDIR=$DATA/plots/logs
-export LOGFIN=$DATA/logs
 mkdir -p $LOGFIN
```

Comment:
@PerryShafran-NOAA @ShelleyMelchior-NOAA  I'm unsure how we can remove export LOGFIN=$DATA/logs and keep the following line (mkdir -p $LOGFIN). Can you please check this for me?

Reply:
This is part of the problem.  I had moved the logfile to a new log directory, but that means that files were not in the original location as denoted by the METplus output.  LOGFIN should not be a directory.  Logs need to be kept in LOGDIR.

Reply:
Removed LOGFIN.